### ISSUE TITLE
Optimize code of doc2vec&keras.py

### ISSUE THREAD

**Issue by liuzulin** [ShuangZhao95/EC601-Cancer-Detection#8]
`from gensim.models import Doc2Vec

Text_INPUT_DIM=300

text_model=None
filename='docEmbeddings_5_clean.d2v'
if os.path.isfile(filename):
    text_model = Doc2Vec.load(filename)
else:
    text_model = Doc2Vec(min_count=1, window=5, size=Text_INPUT_DIM, sample=1e-4, negative=5, workers=4, iter=5,seed=1)
    text_model.build_vocab(sentences)
    text_model.train(sentences, total_examples=text_model.corpus_count, epochs=text_model.iter)
    text_model.save(filename)
    
train_set = np.zeros((train_size, Text_INPUT_DIM))
test_set = np.zeros((test_size, Text_INPUT_DIM))

for i in range(train_size):
    train_set[i] = text_model.docvecs['Text_'+str(i)]

j=0
for i in range(train_size,train_size+test_size):
    test_set[j] = text_model.docvecs['Text_'+str(i)]
    j=j+1
    
print(train_set[0][:50])`
